CHANGELOG.next.asciidoc

```diff
@@ -595,6 +595,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Added cache and connection_errors metrics to status metricset of MySQL module {issue}16955[16955] {pull}19844[19844]
 - Update MySQL dashboard with connection errors and cache metrics {pull}19913[19913] {issue}16955[16955]
 - Add cloud.instance.name into aws ec2 metricset. {pull}20077[20077]
+- Add host inventory metrics into aws ec2 metricset. {pull}20171[20171]
```

Comment:
We will have to add the fields to some `fields.yml` file so they are documented and have a mapping.

Reply:
Yes I'm thinking once we are set on these fields, we can add them to ECS. I can add these fields in `fields.yml` for now and once these fields are in ECS, I assume we don't need them in `fields.yml` anymore?

Reply:
I just added the host fields into fields.yml 😄 Thank you for the review!!